test/IntegrationTests/AspNetTests.cs

```diff
@@ -15,6 +15,7 @@
 // </copyright>
 
 #if NETFRAMEWORK
+using System.Net.Http;
```

Comment:
I didn't expect this to be affected by this change. Do you know why this was changed?

Reply:
Some changes in .NET SDK/compiler. `System.Net.Http` is no longer added to autogenerated headers by .NET Framework compiler.

I have to make similar changes in self-hosted test application.